### PR TITLE
Parametric config for php-fpm

### PR DESCRIPTION
**Pull request by d-r-e**
Related to #101 
Using the commits by @HugeekMcGill 
And it also helps scaling the app for different environments without the predefined/recommended memory requirements.

I'm not having any timeout issues but I am in the necessity to tweak memory resources.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -97,6 +96,13 @@ services:
       - "ADMIN_KEY=${ADMIN_KEY}"
       - "ADMIN_ORG=${ADMIN_ORG}"
       - "GPG_PASSPHRASE=${GPG_PASSPHRASE}"
+      # MISP system settings
+      - "PHP_MEMORY_LIMIT=${PHP_MEMORY_LIMIT:-2048M}"
+      - "PHP_MAX_EXECUTION_TIME=${PHP_MAX_EXECUTION_TIME:-300}"
+      - "PHP_UPLOAD_MAX_FILESIZE=${PHP_UPLOAD_MAX_FILESIZE:-50M}"
+      - "PHP_POST_MAX_SIZE=${PHP_POST_MAX_SIZE:-50M}"
+      - "PHP_MAX_INPUT_TIME:${PHP_MAX_INPUT_TIME}"
+      - "PHP_MAX_EXECUTION_TIME:${PHP_MAX_EXECUTION_TIME}"
```
Comment (d-r-e) [inline docker-compose.yml:105]: The first four vars have a value as default, as they were hardcoded before.

The last two don't need to be modified by default (def val. is -1)

**Comment by ostefano** [comment]
I like the idea of breaking up this issue in two: PHP and NGINX.

**Comment by ostefano** [comment]
Closing because superseded by last build https://github.com/MISP/misp-docker/actions/runs/10538630557

**Comment by d-r-e** [comment]
Thank you!

**Pull request closed without merging**